frc_characterization/newproject/__init__.py

```diff
@@ -297,7 +332,14 @@ def isRotation(units):
     teamNumberEntry.grid(row=2, column=1, sticky="ew")
 
     Label(topFrame, text="Unit Type:", anchor="e").grid(row=2, column=2, sticky="ew")
-    unitChoices = {"Feet", "Inches", "Meters", "Radians", "Rotations"}
+    unitChoices = [
```

Comment:
Why is this in two places in the first place?
Is this file duplicated or something?

Reply:
I believe one of these is in new projects so that users can choose the units that their loggers use, and the other is in the analyzer so that they can convert whatever they measured into some other type of unit if needed. Hopefully this will reduce unit conversion mistakes by the users.

Reply:
Hmmm... does it autodetect the units used in the "new project" / "logger" in the analyser?

Reply:
It does as those units are added onto the json file created by the logger.